src/createShadowDOM.ts

```diff
@@ -15,9 +15,10 @@ const createShadowDOM = (currentChild, pushShadowItem) => {
     setNewChild();
   }
 
+  // @Discuss: Why this is not like this? ==> const setNewChild = (pushShadowItem, validChildren) =>{} and make it in module level.
```

Comment:
@medhatdawoud 
What do you think? :thinking:

Reply:
No need for that, too much customization, again get latest and check code again, it has been changed